app/controllers/packages_controller.rb

```diff
@@ -131,7 +131,7 @@ def url_config_params
   def make_access_urls_plats(url_config_params)
     url_settings = ''
     url_config_params.values.transpose.each do |x|
-      url_settings << x.join("\t") + '|'
+      url_settings << "#{x.join("\t")}|"
```

Comment:
I created an issue to address that the value of `url_settings` can be tested. Right now I think this interpolation should be fine, but there is nothing that tests it, and I do not grok what goes on in the packages_controller enough to write the tests for this PR. See https://github.com/sul-dlss/libsys-webforms/issues/816

Reply:
See https://github.com/sul-dlss/libsys-webforms/pull/817/commits/a86d4116ff083cdb64ad112a6995de2a20b4880d. fixes #816